src/sass/components/govcy-header.scss

```diff
@@ -23,12 +24,13 @@ $govcy-sub-menu-color:#577589;
 	    background-color: $govcy-sub-menu-color;
 	}
 	li {
-		a {
+	a {
 			display: block;
-			padding: 0.9rem !important;
+		    padding: $padding !important;
```

Comment:
Looks too far indented. Not a big deal

Reply:
The indenting is really out in the scss file. It's hard to see what is going on in the file. 
Because we're working as a team now there needs to be a higher emphasis on making code easy to read and understand.

Something like this can do it for you:
https://codebeautify.org/scss-beautifier